build/Publish.targets

```diff
@@ -285,8 +294,11 @@
         RelativeBlobPath="$(BlobFileRelativePathBase)%(FilesToPublish.RelativeBlobPath)"
         ManifestArtifactData="%(FilesToPublish.ManifestArtifactData)"
         Condition=" '%(FilesToPublish.IsDuplicateUpload)' != 'true' " />
-    </ItemGroup>
 
+      <DependencyAssetsDirContent Include="$(DependencyAssetsDir)**" />
+      <DependencyPackagesDirContent Include="$(DependencyPackagesDir)**" />
```

Comment:
Remove these item groups since they appear to be a vestige of debugging code you had for a while